Commons.Music.Midi.DesktopShared/winmm/WinMMNatives.cs

```diff
@@ -60,7 +66,18 @@ public enum MidiOutOpenFlags
 		Event,
 	}
 
-	public delegate void MidiInProc (IntPtr midiIn, uint msg, IntPtr instance, IntPtr param1, IntPtr param2);
+    public enum MidiInMessage : uint
+    {
+        MIM_OPEN = 0x3C1,
+        MIM_CLOSE = 0x3C2,
+        MIM_DATA = 0x3C3,
+        MIM_LONGDATA = 0x3C4,
+        MIM_ERROR = 0x3C5,
+        MIM_LONGERROR = 0x3C6,
+        MIM_MOREDATA = 0x3CC
+    }
```

Comment:
For those "public" enums I would prefer naming members in .NET style i.e. Open, Close, Data...

Reply:
Sure, I'll can that.

I kind of find it annoying though because whenever you run into an issue and need to do a quick google search for how something works and this wrapper has rewritten everything to fit a style. So you need to go an extra step through the original API reference to find what the things are actually called.

Personally I prefer that the wrappers match the original naming 100% so there's no confusion when comparing with the API reference etc. I mean, the functions aren't really matching the C# naming anyway.

But it's your project...

Reply:
I understand that mess. For that kind of readability purpose I use internal enums (that I often generate automatically from C headers using my another project nclang(/samples/PInvokeGenerator) and define public enum types with each corresponding field.